app/controllers/teams/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Teams::UsersController < AuthenticatedController
+   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def remove_confirm
+      @team = team
+      @user = team.users.find_by(id: params[:id])
```

Comment:
**change suggested** We could make this another method similar to the team method below since we do the same thing in two methods.